README.md

```diff
@@ -100,6 +102,31 @@ enabled = true
 period = 60
 ```
 
+## Telemetry
+
+Edgehog sends telemetry from interfaces defined in the
```

Comment:
I might be out of context here, but wouldn't "Edgehog sends telemetry _data_ from interfaces" fit better?